### PR TITLE
[mod_say_nb] add norwegian say module

### PR DESCRIPTION
**Pull request by lukas-wg2**
Going to need some input from a real norwegian...

### CODE REVIEW

**Comment by lukas-wg2** [comment]
TODO: figure out what grammar rules i need

**Comment by lukas-wg2** [comment]
Basing all changes on this document: https://www.riksmalsforbundet.no/grammatikk/kapittel-7-tallord/

File: src/mod/say/mod_say_nb/Makefile.am [hunk 1]
```diff
@@ -0,0 +1,8 @@
+include $(top_srcdir)/build/modmake.rulesam
+MODNAME=mod_say_nb
+
+mod_LTLIBRARIES = mod_say_nb.la
+mod_say_sv_la_SOURCES  = mod_say_nb.c
```
Comment (pmodin-wg2) [inline src/mod/say/mod_say_nb/Makefile.am]: `mod_say_sv_`...

File: src/mod/say/mod_say_nb/mod_say_nb.c [hunk 2]
```diff
@@ -0,0 +1,676 @@
...
+				if (method == SSM_COUNTED) {
+					switch_say_file(sh, "digits/r-%d", c);
+				} else {
+					switch_say_file(sh, "digits/%d", c);
+				}
+			}
+		}
+	}
+
+        if (what && (a || b || c)) {
```
Comment (pmodin-wg2) [inline src/mod/say/mod_say_nb/mod_say_nb.c:143]: don't mix indent. Goes for all files in this PR

File: src/mod/say/mod_say_nb/mod_say_nb.c [hunk 3]
```diff
@@ -0,0 +1,676 @@
...
+			!= SWITCH_STATUS_SUCCESS) {						\
+			return tstatus;								\
+		}										\
+		say_args->method = smeth; say_args->type = stype;				\
+	}											\
+
+static switch_status_t play_group(switch_say_method_t method, switch_say_gender_t gender, int a, int b, int c, char *what, switch_say_file_handle_t *sh)
+{
+	/*
+	 * Norweigian, like swedish, have gendered numbers for the number one. Utrum is 'en' while neutrum is 'ett'. They also have 'et' but we'll probably
```
Comment (pmodin-wg2) [inline src/mod/say/mod_say_nb/mod_say_nb.c]: Reflow, this looks like java :smirk_cat: 

goes for all lines in this file.

File: src/mod/say/mod_say_nb/mod_say_nb.c [hunk 4]
```diff
@@ -0,0 +1,676 @@
...
+		}
+	} else {
+		switch_say_file(sh, "digits/0");
+		switch_say_file(sh, "currency/ore");
+	}
+
+	return SWITCH_STATUS_SUCCESS;
+}
+
+static switch_status_t say_ip(switch_say_file_handle_t *sh, char *tosay, switch_say_args_t *say_args)
```
Comment (pmodin-wg2) [inline src/mod/say/mod_say_nb/mod_say_nb.c]: do we need these unused functions? do they work?

File: src/mod/say/mod_say_nb/mod_say_sv.2017.vcxproj [hunk 5]
```diff
@@ -0,0 +1,139 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (pmodin-wg2) [inline src/mod/say/mod_say_nb/mod_say_sv.2017.vcxproj]: is this file tested or just copied?

File: src/mod/say/mod_say_nb/Makefile.am (hunk 1 above)
Comment (pmodin-wg2) [reply src/mod/say/mod_say_nb/Makefile.am]: the la files are not included, does it work anyways?

File: src/mod/say/mod_say_nb/mod_say_nb.c [hunk 6]
```diff
@@ -0,0 +1,676 @@
...
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * The Initial Developer of the Original Code is
+ * Anthony Minessale II <anthm@freeswitch.org>
+ * Portions created by the Initial Developer are Copyright (C)
+ * the Initial Developer. All Rights Reserved.
+ *
+ * Contributor(s):
+ *
+ * Anthony Minessale II <anthm@freeswitch.org>
```
Comment (pmodin-wg2) [inline src/mod/say/mod_say_nb/mod_say_nb.c]: replace this list with just yourself :smile_cat:

**Review by pmodin-wg2 — COMMENTED** [review]
please use fixup commits

File: src/mod/say/mod_say_nb/mod_say_nb.c (hunk 3 above)
Comment (pmodin-wg2) [reply src/mod/say/mod_say_nb/mod_say_nb.c]: A good measure is soft limit at 80, hard at 120 chars.

File: src/mod/say/mod_say_nb/mod_say_sv.2017.vcxproj (hunk 5 above)
Comment (lukas-wg2) [reply src/mod/say/mod_say_nb/mod_say_sv.2017.vcxproj]: copied, can remove

File: src/mod/say/mod_say_nb/mod_say_nb.c (hunk 2 above)
Comment (lukas-wg2) [reply src/mod/say/mod_say_nb/mod_say_nb.c:143]: https://www.youtube.com/watch?v=2g5Hz17C4is

**Pull request merged**